src/utils/mock_data/pullData.js

```diff
@@ -0,0 +1,98 @@
+const { Octokit } = require('@octokit/core');
+const path = require('path');
+const dotenv = require('dotenv');
+const usersInfos = require('./users');
+
+// range: [min, max)
+function getRandomInt(min, max) {
+    min = Math.ceil(min);
+    max = Math.floor(max);
+    return Math.floor(Math.random() * (max - min) + min);
+}
+  
+
+dotenv.config({ path: path.resolve(__dirname, '../../../.env') });
+
+function repoFormat(repo, id, creator) {
+    let project = {};
+    project.id = id;
+    project.name = repo.name.replace(new RegExp('-', 'g'), ' ');
+    project.creator = creator;
+    project.description = repo.description;
+    project.skills = repo.languages;
+    project.tags = repo.topics.map(topic => topic.replace(new RegExp('-', 'g'),' '));
+    project.links = [{ type: 'GitHub', link: repo.html_url }]
+
+    return project;
+}
+
+
+/**
+ * @param {string} query - search query
+ */
+const fetchRepos = async (query) => {
+
+    const octokit = new Octokit({auth: process.env.GITAUTH});
```

Comment:
Do we still need to have an env for GITAUTH?

Reply:
I already put the data in the GitHub repository itself, so we shouldn't have to run the script anymore, but if anyone would want to use the script to pull fresh data, they would need it.